sdk/__test__/util.spec.ts

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios'
+import { fetchJwt } from '../utils';
+
+jest.mock("axios");
+const mockedAxios = axios as jest.Mocked<typeof axios>;
```

Comment:
will this end up using the real instance of axios? this line is just casting the real object (not creating a mock afaik). ideally we shouldnt import axios in unit tests

Reply:
Updated to use mock